src/main/java/flaxbeard/steamcraft/integration/BotaniaIntegration.java

```diff
@@ -129,6 +129,8 @@ public static Multimap addModifiers(Multimap map, ItemStack stack, int armorType
                     break;
                 case 4:
                     hp = 2;
+                default: 
+                	break;
```

Comment:
Online editor and view show different spaces! Editor window is OK But View is NOK!